dashboard/src/pages/hardwareDetails/Tabs/Tests/TestsTab.tsx

```diff
@@ -115,6 +128,11 @@ const TestsTab = ({ tests, trees, hardwareId }: TTestsTab): JSX.Element => {
               configStatusCounts={tests.configs}
               diffFilter={diffFilter}
             />
+            <MemoizedPlatformsCard
```

Comment:
because everything is in the same div, everything is on the same column can you spread the cards between two divs here?